client/src/utils/HTTPClient.js

```diff
@@ -0,0 +1,19 @@
+const Client = {
+  request: async (endpoint, method = 'GET', body) => {
```

Comment:
why are we including a body with a get request? Also what is the benefit to using this request compared to using the feathers client for fetch requests?

If you want to do it this way that's fine, but you'll eventually also have to write out methods for post, put, patch, and delete requests as well.

Reply:
or wait is the body variable for the response?

Reply:
You wouldn't be sending a body with a get request. That syntax is just a default if one isn't specified, which gives it flexibility to be used as a post or whatever request you need it to be.

If you want to make a post request to an endpoint, you have to specify. If you don't specify, it'll default to a get request.

Ideally I'd like to at least test the backend with something that I know works before switching over to feathers-client, but I'll also add support for the feathers client